kolibri/core/assets/src/views/sync/SelectDeviceModalGroup/DeviceConnectingModal.vue

```diff
@@ -80,17 +84,17 @@
       connectionFailureMessage() {
         const connectionStatus = '';
         switch (connectionStatus) {
-          case 'Unknown':
-          case 'ConnectionFailure':
-            return this.isInternetDevice
-              ? this.$tr('internetDisconnected')
-              : this.$tr('connectionFailure', { name: this.deviceName });
-          case 'ResponseTimeout':
-            return this.$tr('connectionTimedOut', { name: this.deviceName });
-          case 'InvalidResponse':
-            return this.$tr('deviceIsNotKolibri', { name: this.deviceName });
-          default:
-            return this.$tr('genericConnectionError');
+        case 'Unknown':
+        case 'ConnectionFailure':
+          return this.isInternetDevice
+            ? this.$tr('internetDisconnected')
+            : this.$tr('connectionFailure', { name: this.deviceName });
+        case 'ResponseTimeout':
+          return this.$tr('connectionTimedOut', { name: this.deviceName });
+        case 'InvalidResponse':
+          return this.$tr('deviceIsNotKolibri', { name: this.deviceName });
+        default:
+          return this.$tr('genericConnectionError');
```

Comment:
Not sure what prettier / linting rule is forcing this but it seems not ideal. I don't see any reason why the cases inside a switch should not be indented similarly to how you indent something in a function or `if` block